### PR TITLE
586 redirect to the start page

### PR DESCRIPTION
**Pull request by kacperpONS**
# Motivation and Context

Solved the issue with "Return to previous page" link not showing after accepting the cookies on the cookies page.
Added link to the logo.
 
# What has changed

Solved the issue with the link not showing by changing the Referrer Policy header.

Added link to the logo. 

Renamed domain_url_en to domain_url

# How to test?

Test that the logo links to the start page

# Links

Trello card:
https://trello.com/c/1Hiu4GbK/586-rhui-redirect-to-the-start-page-after-saving-changes-on-cookies-page

# Screenshots (if appropriate):

### CODE REVIEW

File: app/security.py [hunk 1]
```diff
@@ -47,7 +47,7 @@
     'X-Content-Security-Policy': CSP,
     'X-Frame-Options': 'DENY',
     'X-Content-Type-Options': 'nosniff',
-    'Referrer-Policy': 'no-referrer',
+    'Referrer-Policy': 'strict-origin-when-cross-origin',
```
Comment (LukeLoze1) [inline app/security.py:50]: Is this the most secure it can be set to?

Comment (davidmort) [reply app/security.py:50]: there are more "secure" options, but this is a sensible option and also allows the sharing of the full referrer with our domain which some of the design system needs, this is a good guide on how to choose the right option: https://web.dev/referrer-best-practices/#:~:text=Consider%20setting%20a%20referrer%20policy,an%20extra%20layer%20of%20security.

File: run.py [hunk 2]
```diff
@@ -7,5 +7,5 @@
 
 if __name__ == '__main__':
     from app.app import create_app
-    app = create_app()
+    app = create_app(os.environ['APP_SETTINGS'])
```
Comment (AdamHawtin) [inline run.py:10]: `APP_SETTINGS` has been removed as an environment variable in #79 and shouldn't be used here.

I see here that I missed some references to it though, so I've also fixed those separately in #85.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

File: run.py (hunk 2 above)
Comment (kacperpONS) [reply run.py:10]: Done 👍

Comment (kacperpONS) [reply run.py:10]: Small note:
Removing it causes an error when creating the app from config, but it's solved in #85

**Review by LukeLoze1 — APPROVED** [review]

File: app/templates/base-en.html [hunk 3]
```diff
@@ -87,7 +87,7 @@
 {%- set pageConfig = {
     'title': page_title_value,
     'header': {
-        'logoHref': 'https://www.ons.gov.uk/',
+        'orgLogoHref': domain_url + '/en/start/',
```
Comment (AdamHawtin) [inline app/templates/base-en.html:90]: I think we still want this top logo banner to link to the organisation homepage, rather than our own homepage.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
The cookies page changes all look good to me, I don't think we want to change the top banner logo link destination though.

**Pull request merged**